docs/examples.md

```diff
@@ -389,6 +389,22 @@ test_task:
   test_script: yarn run test
 ```
 
+### Yarn 2
+
+Yarn 2 (also known as Yarn Berry), has a different way of caching packages.
+It stores all packages inside the repository, as files that should be committed.
```

Comment:
You should probably still cache the folder 🤔

Reply:
Yarn 2 literally puts all the package zipballs in the repository, e.g. https://github.com/rdilweb/rdil.rocks/tree/refactoring/.yarn/cache

Reply:
That's not a good idea for a Git repository. It will blow out the size of it since all version will be stored. You should add `.yarn` to `.gitignore` and cache it instead.